Source/Particles/ElementaryProcess/QEDInternals/QedWrapperCommons.H

```diff
@@ -36,4 +40,38 @@
 #define PXRMP_FORCE_INLINE AMREX_FORCE_INLINE
 //_________________________
 
+
+/**
+* Thin Wrapper around amrex::Gpu::DeviceVector<Real> to make it usable
+* as the building block of the lookup tables when managed memory is not used.
+*
+* @tparam Real the floating point type to be used
+*/
+template <typename Real>
+class AmrexDeviceVectorWrapper : public amrex::Gpu::DeviceVector<Real>
+{
+    using AGDV = amrex::Gpu::DeviceVector<Real>;
+    public:
+
+    /**
+    * All the arguments passed to AmrexDeviceVectorWrapper constructor
+    * are forwarded to amrex::Gpu::DeviceVector constructor.
+    *
+    * @tparam Args the constructor arguments
+    */
+    template<typename... Args>
+    AmrexDeviceVectorWrapper(Args&&... args) : AGDV(std::forward<Args>(args)...)
+    {}
+
+    /**
+    * This function may be called in some places inside picsar_table classes.
+    * It forces a copy of the CPU data to the GPU
+    *
+    */
+    void pxr_sync()
+    {
+        amrex::Gpu::synchronize();
```

Comment:
Is stream sync sufficient?
```suggestion
        amrex::Gpu::Device::streamSynchronize();
```